.storybook/main.cjs

```diff
@@ -138,35 +138,37 @@ module.exports = {
       // Use asset svg and svgr in same project using resourceQuery
       // https://react-svgr.com/docs/webpack/#use-svgr-and-asset-svg-in-the-same-project
       {
-        type: 'asset/inline',
         test: /\.svg$/,
-        resourceQuery: /url/,
-      },
-      {
-        test: /\.svg$/,
-        issuer: /\.js?$/,
-        exclude: /\?url$/,
-        use: [
+        oneOf: [
```

Comment:
Now we only use one loader. If a `url` resource query is found on the import (i.e. `./icons/arrow_circle.svg?url`) then that asset will be loaded in using the `asset/inline` asset module.

Importing both in the same file will import one as a reach component, and the other as a data uri ✨ 
```js
import arrowCircle from './icons/arrow_circle.svg?url';
import ArrowCircle from './icons/arrow_circle.svg';
```
<img width="504" alt="image" src="https://user-images.githubusercontent.com/22185279/154130381-a13beba5-94ae-4255-9fc7-83607ce38d66.png">